examples/query/synth_data_llm_filter.py

```diff
@@ -0,0 +1,169 @@
+import sycamore
+from sycamore.functions import HuggingFaceTokenizer
+from sycamore.llms import OpenAI, OpenAIModels
+from sycamore.transforms.embed import SentenceTransformerEmbedder
+import os
+from opensearchpy import OpenSearch
+import argparse
+import random
+from sycamore.data import Document, Element
+from sycamore.docset import DocSet
+
+argparser = argparse.ArgumentParser(prog="synth_data_llm_filter")
+argparser.add_argument("--oshost", default=None)
+argparser.add_argument("--osport", default=9200)
+argparser.add_argument("--maxdocs", default=10)
+argparser.add_argument("--maxelems", default=5)
+argparser.add_argument("--index", default=None)
+args = argparser.parse_args()
+
+# The OpenSearch index name to populate.
+if args.index is not None:
+    INDEX = args.index
+else:
+    INDEX = "synth_10"
+
+# The OpenSearch instance to use.
+if os.path.exists("/.dockerenv"):
+    opensearch_host = "opensearch"
+    print("Assuming we are in a Sycamore Jupyter container, using opensearch for OpenSearch host")
+else:
+    opensearch_host = "localhost"
+    print("Assuming we are running outside of a container, using localhost for OpenSearch host")
+
+opensearch_port = args.osport
+
+os_client_args = {
+    "hosts": [{"host": opensearch_host, "port": opensearch_port}],
+    "http_compress": True,
+    "http_auth": ("admin", "admin"),
+    "use_ssl": True,
+    "verify_certs": False,
+    "ssl_assert_hostname": False,
+    "ssl_show_warn": False,
+    "timeout": 120,
+}
+
+os_client = OpenSearch(**os_client_args)  # type: ignore
+
+index_settings = {
+    "body": {
+        "settings": {"index.knn": True, "number_of_shards": 5, "number_of_replicas": 1},
+        "mappings": {
+            "properties": {
+                "embedding": {
+                    "dimension": 384,
+                    "method": {
+                        "engine": "faiss",
+                        "space_type": "l2",
+                        "name": "hnsw",
+                        "parameters": {},
+                    },
+                    "type": "knn_vector",
+                }
+            }
+        },
+    }
+}
+
+# The number of documents to generate.
+max_docs = int(args.maxdocs)
+
+# The maximum number of elements in each document.
+max_elems = int(args.maxelems)
+
+
+def make_element(text):
```

Comment:
I'd remove the docstring and just call this element_from_text. If you keep the docstring for something this simple, I expect it for all the functions.